README.md

```diff
@@ -88,7 +88,14 @@ built-in LSP client.
   - `inlay_hints_highlight` (string): Highlight group used for inlay hints.
     Defaults to "Comment".
 
-- Filter `tsserver` diagnostics
+  - `inlay_hints_priority` (number): Priority of the hint extmarks. Change
```

Comment:
Could we also document that `updatetime` affects the time between updating inlay hints?